BIBER_COMPILER check

Some system has LATEX_COMPILER only, but do not have BIBER_COMPILER.
It will not work for the update in "usermanuals/DD4hep" in Update DD4hepManual #362.
Would you please add this check?
Or you have better solution for the BIBER_COMPILER check.

doc/CMakeLists.txt

    IF(BIBER_COMPILER)
        ADD_SUBDIRECTORY(usermanuals/DD4hep)
    ENDIF(BIBER_COMPILER)

**CMake Error at cmake/LATEX.cmake:1585 (message):
  I need the biber command.**


```shell
COMPILER_PATH=/afs/cern.ch/sw/lcg/contrib/gcc/4.9.3/x86_64-slc6/lib/gcc/x86_64-unknown-linux-gnu/4.9.3
CC=gcc
----- END OF ENVIRONMENT SNAPSHOT -----
loading initial cache file /scratch/nbuilds/gcc49/2018-04-13/ILCSoft.cmake
-- CMAKE_INSTALL_PREFIX is /scratch/nbuilds/gcc49/2018-04-13/DD4hep/HEAD - overwrite with -D CMAKE_INSTALL_PREFIX
-- Found Doxygen: /usr/bin/doxygen (found version "1.6.1") 
-- Found LATEX: /usr/bin/latex   
-- Found LaTeX compiler.
-- Found UnixCommands: /bin/bash  
-- I could not find the htlatex command.
-- I could not find the biber command.
CMake Error at cmake/LATEX.cmake:1585 (message):
  I need the biber command.
Call Stack (most recent call first):
  cmake/LATEX.cmake:1892 (add_latex_targets_internal)
  doc/usermanuals/DD4hep/CMakeLists.txt:2 (ADD_LATEX_DOCUMENT)


-- The CXX compiler identification is GNU 4.9.3
-- Check for working CXX compiler: /afs/cern.ch/sw/lcg/contrib/gcc/4.9.3/x86_64-slc6/bin/g++
-- Check for working CXX compiler: /afs/cern.ch/sw/lcg/contrib/gcc/4.9.3/x86_64-slc6/bin/g++ -- works
```

--- a/doc/CMakeLists.txt
+++ b/doc/CMakeLists.txt
@@ -24,7 +24,7 @@ ENDIF(DOXYGEN_FOUND)
 FIND_PACKAGE(LATEX)
 
 # Only add target if we have LaTeX
-IF(LATEX_COMPILER)
+IF(LATEX_COMPILER AND BIBER_COMPILER)
     MESSAGE(STATUS "Found LaTeX compiler.")
     SET(LATEX_COMPILER_FLAGS "-shell-escape" "-interaction nonstopmode" "-halt-on-error" "-file-line-error" CACHE STRING "Flags passed to latex.")
     INCLUDE(${CMAKE_SOURCE_DIR}/cmake/LATEX.cmake)
@@ -39,6 +39,6 @@ IF(LATEX_COMPILER)
     ADD_SUBDIRECTORY(usermanuals/DDG4)
     ADD_SUBDIRECTORY(usermanuals/DDRec)
     
-ELSE(LATEX_COMPILER)
-    MESSAGE(STATUS "No LaTeX found, cannot compile user manual.")
-ENDIF(LATEX_COMPILER)
+ELSE()
+    MESSAGE(STATUS "No LaTeX/Biber found, cannot compile user manual.")
+ENDIF()

